cloud/deployment/deployment_test.go

```diff
@@ -1288,6 +1288,7 @@ func TestUpdate(t *testing.T) { //nolint
 		// success with hybrid type
 		err := Update("", "", ws, "", "", "", CeleryExecutor, "", "", "", "", "", "", "", 0, 0, workerQueueRequest, hybridQueueList, newEnvironmentVariables, true, mockCoreClient, mockPlatformCoreClient)
```

Comment:
Can we add a comment up here w.r.t to what are we updating? 

Eventually would want to get rid of that huge list of arguments and have defined input type, but we can do that later.

Reply:
added comments